src/libutil/util.cc

```diff
@@ -1472,15 +1473,12 @@ string base64Encode(std::string_view s)
 
 string base64Decode(std::string_view s)
 {
-    bool init = false;
-    char decode[256];
-    if (!init) {
-        // FIXME: not thread-safe.
-        memset(decode, -1, sizeof(decode));
+    static std::once_flag flag;
+    std::call_once(flag, [](){
+        base64DecodeChars = { (char)-1 };
```

Comment:
If you expect that fills `base64DecodeChars` with `-1`'s, that is wrong

the code effectively does `base64DecodeChars[0] = -1`